be-library/service/reminder.go

```
@@ -1173,51 +1176,187 @@ func (s *ReminderService) subscriptionStillCurrent(ctx context.Context, expected
 		if err := ctx.Err(); err != nil {
 			launchErr = err
 			break
 		}
-		row := rows[i]
 		select {
 		case sem <- struct{}{}:
 		case <-ctx.Done():
 			launchErr = ctx.Err()
-			launching = false
-			continue
+			break
```

golangci-lint (2.13.2)

[error] SA4011: ineffective break statement. Did you mean to break out of the outer loop?

(staticcheck)